pkg/collectors/service.go

```diff
@@ -53,12 +53,40 @@ var (
 		nil,
 	)
 
+	descServiceExternalName = metrics.NewMetricFamilyDef(
```

Comment:
`descServiceExternalName` is defined here, but not used anywhere. Am I missing something?